pkg/segment/structs/segstructs.go

```diff
@@ -439,6 +439,7 @@ type NodeResult struct {
 	CurrentSearchResultCount    int
 	AllSearchColumnsByTimeRange map[string]bool
 	FinalColumns                map[string]bool
+	GlobalErrors                map[string]*ErrorInfo // maps global error from error message -> error stats
```

Comment:
Should this be merged with the `ErrList` field, or should they stay separate? If they stay separate, I think it would be good to move one of these fields so they're both adjacent

Reply:
hmm.. ErrList should eventually be deprecated and we should move to `GlobalErrors`, primarily because ErrList is an array and we append too many of the same errors to it, it will get large. 

And maybe we should call this as `GlobalSearchErrors`
Agree with moving these two adjacent and adding a note about eventually replacing ErrList to this new var.